addon/services/cookies.js

```diff
@@ -67,10 +67,16 @@ export default Service.extend({
   },
 
   write(name, value, options = {}) {
+    if (isEmpty(options.encodeValue)) {
+      options.encodeValue = true;
+    }
     assert('Cookies cannot be set to be HTTP-only as those cookies would not be accessible by the Ember.js application itself when running in the browser!', !options.httpOnly);
     assert("Cookies cannot be set as signed as signed cookies would not be modifyable in the browser as it has no knowledge of the express server's signing key!", !options.signed);
     assert('Cookies cannot be set with both maxAge and an explicit expiration time!', isEmpty(options.expires) || isEmpty(options.maxAge));
-    value = this._encodeValue(value);
+    assert('The encodeValue option must be a boolean!', typeof options.encodeValue === 'boolean');
```

Comment:
I don't think we need that - if someone want to set a truthy value I'd say that's fine.